src/pymodule/dense/morphology/connections.py

```diff
@@ -58,16 +58,16 @@ def generate_network(source_neurons=None, target_neurons=None,
 
     .. math::
 
-        n_{s, p} = \rho_s \cdot A_I
+        n_{s, p} = \\rho_s \\cdot A_I
```

Comment:
let's make it simpler (so we can also copy paste from it), add ``r`` in from of the ``"""`` on line 51, this way it makes it a raw string and we can directly use ``\`` rather than ``\\``